config/application.rb

```diff
@@ -46,5 +46,10 @@ class Application < Rails::Application
 
     # Emails
     config.email_test_address = ENV["EMAIL_TEST_ADDRESS"]
+
+    # Grace window value in days from when a registration expires. During the
+    # grace window users will still be permitted to renew their registration.
+    config.registration_renewal_grace_window = ENV["REGISTRATION_RENEWAL_GRACE_WINDOW"] || 30
+    config.first_renewal_email_reminder_days = ENV["FIRST_RENEWAL_EMAIL_REMINDER_DAYS"]
```

Comment:
As we've used defaults for other config values, I think it would be fine to also have one for the email reminder days

```suggestion
    config.first_renewal_email_reminder_days = ENV["FIRST_RENEWAL_EMAIL_REMINDER_DAYS"] || 30
```

What do you think?

Reply:
I am not sure about that. I would prefer stuff to fail rather than work and then we send the email with a token that expires at the wrong time